src/docs/product/releases/health/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Health
+sidebar_order: 40
+redirect_from:
+  - /workflow/releases/health/
+  - /product/releases/health/
+description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
+---
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session-data). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+## Enabling Release Health
+
+Many SDKs automatically manage the start and end of sessions when the SDK is initialized, but release health configuration is key to ensuring you are receiving data that is useful, so check our information on [which SDKs support release health](/product/releases/health/setup/) for links to configuring the SDK, as needed.
+
+## Data Filtering
+
+Sentry provides [inbound data filters](/product/error-monitoring/filtering/) to filter data into your project. Data filters are only partially supported for session events.
+
+## What Release Health Monitors
+
+Once you configure your SDK, Sentry connects the metrics to the specific release of your application and the associated code.
+
+### Session Data
+
+The primary component Sentry uses to monitor health is a _session_, which represents the interaction between the user and the application.
+
+A session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground.
```

Comment:
Let's keep this on our backlog for later: The definition of what a session is varies by platform. The original content was written for mobile, so it may no longer be accurate, especially for JavaScript. 

I would propose we follow up with this and give a general notion of what a session is (along the lines of "a continuous interaction of a user with your application") and then give examples for various platforms.

Reply:
Agreed. I've opened https://github.com/getsentry/sentry-docs/issues/2892 to track.